### ISSUE TITLE
[FEAT]: Expert Train & Inference for Mustard dataset with Mistral-7B

### ISSUE THREAD

**Issue by Jasonqi146** [lwaekfjlk/mmodel-soup#92]
### Description

Use Mustard training split of R, U, and S to train model Mistral-7B seperately and save to

Sarc R finetuned Mistral-7B
Sarc U finetuned Mistral-7B
Sarc S finetuned Mistral-7B
Then record the performance of the three models on the test set.
